iphone/cli/commands/_buildModule.js

```diff
@@ -0,0 +1,803 @@
+	this.tiSymbols = {};
+	this.documentation = [];
+	this.metaData = [];
+	this.metaDataFile = path.join(this.projectDir, 'metadata.json');
+	this.manifestFile = path.join(this.projectDir, 'manifest');
+	this.titaniumIosSdkPath = appc.fs.resolvePath(__dirname, '..', '..');
+	this.templatesDir = path.join(this.titaniumIosSdkPath, 'templates');
+	this.assetsTemplateFile = path.join(this.templatesDir, 'module', 'default', 'template', 'iphone', 'Classes', '{{ModuleIdAsIdentifier}}ModuleAssets.m.ejs');
+
+	this.universalBinaryDir = path.join(this.projectDir, 'build');
+
+	['assets', 'documentation', 'example', 'platform'].forEach(function (folder) {
+		var dirName = folder+'Dir';
+		this[dirName] = path.join(this.projectDir, folder);
+		if (!fs.existsSync(this[dirName])) {
+			this[dirName] = path.join(this.projectDir, '..', folder);
+		}
+	}, this);
+
+	this.licenseDefault = "TODO: place your license here and we'll include it in the module distribution";
+	this.licenseFile = path.join(this.projectDir, 'LICENSE');
+	if (!fs.existsSync(this.licenseFile)) {
+		this.licenseFile = path.join(this.projectDir, '..', 'LICENSE');
+	}
+
+	this.tiXcconfig = {};
+	this.tiXcconfigFile = path.join(this.projectDir, 'titanium.xcconfig');
+
+	this.moduleXcconfigFile = path.join(this.projectDir, 'module.xcconfig');
+
+	next();
+};
+
+iOSModuleBuilder.prototype.loginfo = function (next) {
+	this.logger.debug(__('Titanium SDK iOS directory: %s', this.platformPath.cyan));
+	this.logger.info(__('Project directory: %s', this.projectDir.cyan));
+	this.logger.info(__('Module ID: %s', this.manifest['moduleid'].cyan));
+
+	next();
+};
+
+iOSModuleBuilder.prototype.dirWalker = function (currentPath, callback) {
+	var files = fs.readdirSync(currentPath);
+	for (var i in files) {
+		var currentFile = path.join(currentPath, files[i]);
+		var stats = fs.statSync(currentFile);
+
+		if (stats.isFile()) {
+			callback(currentFile);
+		} else if (stats.isDirectory()) {
+			this.dirWalker(currentFile, callback);
+		}
+	}
+};
+
+iOSModuleBuilder.prototype.processLicense = function (next) {
+	if (fs.existsSync(this.licenseFile)) {
+		if (fs.readFileSync(this.licenseFile).toString().indexOf(this.licenseDefault) != -1) {
+			this.logger.warn(__('Please update the LICENSE file with your license text before distributing.'));
+		}
+	}
+
+	next();
+};
+
+iOSModuleBuilder.prototype.processTiXcconfig = function (next) {
+	var re = /^(\S+)\s*=\s*(.*)$/,
+		bindingReg = /\$\(([^$]+)\)/g,
+		match,
+		bindingMatch;
+
+	if (fs.existsSync(this.tiXcconfigFile)) {
+		fs.readFileSync(this.tiXcconfigFile).toString().split('\n').forEach(function (line) {
+			match = line.match(re);
+			if (match) {
+				var keyList = [],
+					value = match[2].trim();
+
+				bindingMatch = bindingReg.exec(value);
+				if (bindingMatch != null) {
+					while (bindingMatch != null) {
```

Comment:
This and preceding line, you should use !== unless you have a reason to be using !=